### PR TITLE
[Tidy] Move `kpi` example dashboard to HF

### PR DESCRIPTION
**Pull request by huong-li-nguyen**
## Description
- Remove `kpi` folder from examples
- Update README.md with new screenshot
- Create README.md for `examples` folder

HF Link: https://huggingface.co/spaces/vizro/demo-kpi - now public 🥳 

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-core/examples/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Please note that this folder contains only example dashboards that are still **in development**.
+
+To view a comprehensive list of available demos, please visit our [examples gallery](http://vizro.mckinsey.com/).
+There, you can explore a wide range of dashboards and applications created with Vizro.
+
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/tidy/remove-kpi-dashboard/.github/images/vizro_examples_gallery.png" width="800"/>
+
+For a curated list of example dashboards, check out our [dashboard collection on Huggingface](https://huggingface.co/collections/vizro/vizro-official-gallery-66697d414646eeac61eae6de).
+
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/tidy/remove-kpi-dashboard/.github/images/huggingface_collection.png" width="800"/>
```
Comment (huong-li-nguyen) [inline vizro-core/examples/README.md:12]: Replace `tidy/remove-kpi-dashboard` with `main` before merge

File: vizro-core/examples/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+### Vizro examples gallery
+
+Please note that this folder contains only example dashboards that are still **in development**.
+
+To view a comprehensive list of available demos, please visit our [examples gallery](http://vizro.mckinsey.com/).
+There, you can explore a wide range of dashboards and applications created with Vizro.
+
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/tidy/remove-kpi-dashboard/.github/images/vizro_examples_gallery.png" width="800"/>
```
Comment (huong-li-nguyen) [inline vizro-core/examples/README.md:8]: Replace tidy/remove-kpi-dashboard with main before merge

**Review by maxschulz-COL — APPROVED** [review]
👍 Loks good to me :)

**Review by petar-qb — APPROVED** [review]
Looks good!

**Pull request merged**